deploy/nginx/start.sh

```diff
@@ -3,4 +3,6 @@ set -e
 
 cd /app/dist
 
+sed -i "s/\/BASE_PLACEHOLDER\//$RUNTIME_BASE_URL/g" ./*
```

Comment:
```suggestion
sed -i "s#/BASE_PLACEHOLDER/#$RUNTIME_BASE_URL#g" ./*
```
sed 可以用其他字符作为分隔符 比如 `#` 你试试看呢